x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -56,6 +58,12 @@ import { HostIsolationExceptionsApiClient } from './pages/host_isolation_excepti
 import { IconAssetCriticality } from '../common/icons/asset_criticality';
 
 const categories = [
+  {
+    label: i18n.translate('xpack.securitySolution.appLinks.category.investigations', {
+      defaultMessage: 'Investigations',
+    }),
+    linkIds: [SecurityPageName.notesManagement],
+  },
```

Comment:
IMO, this category doesn't have the highest priority and it now pushes some other functionality below the fold. Wdyt about moving this to the end of the list?

Reply:
sure, I don't mind :)